docs/connect.html

```diff
@@ -174,7 +176,7 @@ <h4><a id="connect_transforms" href="#connect_transforms">Transformations</a></h
         <li>InsertField - Add a field using either static data or record metadata</li>
         <li>ReplaceField - Filter or rename fields</li>
         <li>MaskField - Replace field with valid null value for the type (0, empty string, etc)</li>
-        <li>ValueToKey</li>
+        <li>ValueToKey - Replace the record key with a new key formed from a subset of fields in the record value</li>
```

Comment:
Add a one line description for `ValueToKey` transformation to be consistent with others. The description is copied from the detailed explanation below.
before:
![image](https://user-images.githubusercontent.com/43372967/82002297-04e82080-9690-11ea-9e1b-6cb218b0593c.png)

description source:
![image](https://user-images.githubusercontent.com/43372967/82002385-38c34600-9690-11ea-8733-19ed835a5a3f.png)